x/ccv/provider/keeper/relay.go

```diff
@@ -419,16 +426,20 @@ func (k Keeper) HandleSlashPacket(ctx sdk.Context, chainID string, data ccv.Slas
 	)
 
 	// Obtain validator from staking keeper
-	validator, found := k.stakingKeeper.GetValidatorByConsAddr(ctx, providerConsAddr.ToSdkConsAddr())
+	validator, err := k.stakingKeeper.GetValidatorByConsAddr(ctx, providerConsAddr.ToSdkConsAddr())
+	if err != nil && errors.Is(err, stakingtypes.ErrNoValidatorFound) {
+		k.Logger(ctx).Error("validator not found", "validator", providerConsAddr.String())
+		return
+	}
```

Comment:
What happens if the error is a different type of error? Shouldn't we also `return` in such a case and also log something?

Reply:
Nice catch! Removed the condition on the error type to always log stg and return if the returned error isn't nil.